takwimu/views.py

```diff
@@ -372,10 +372,13 @@ def get(self, request, *args, **kwargs):
                                            operator=operator,
                                            query_type='phrase_prefix',
                                            quert_fields=['title'])
-        suggestions = hits[:5] if len(hits) > 5 else hits
-        results = [{"title": suggestion.get('title')} for suggestion in suggestions]
+
+        results = [hit.get("title") for hit in hits]
+        # remove duplicates
+        results = list(set(results))
+        suggestions = results[:5] if len(results) > 5 else results
```

Comment:
aren't we allowing the user to scroll through the results as per design ? cc @kilemensi

Reply:
Yes we're. I've changed this 10 on suggestion and 50 on actual search @karimkawambwa